discoverecs.py

```diff
@@ -457,11 +461,11 @@ def task_info_to_targets(task_info):
 
 
 class Main:
-    def __init__(self, directory, interval, default_scrape_interval_prefix):
+    def __init__(self, directory, interval, default_scrape_interval_prefix, cluster):
```

Comment:
This would be more useful as a list so multiple clusters can be used

Reply:
I agree and originally planned to do this, but my Python skills are very poor and I have a hard time testing the script. I will try my best to make the appropriate changes. Thanks for the help.